### DISCUSSION TITLE
Backup altervista gratuito

### DISCUSSION THREAD

**Discussion by luigiadoc** [iisgiua/giuaschool#750] [Richieste di aiuto]
Salve, usufruendo di un dominio altervista gratuito non è garantito il backup settimanale e, pertanto, vorrei chiedere prima di effettuare il passaggio al nuovo anno scolastico (premetto che ho fatto tutti gli scrutini anche del giudizio sospeso con successo salvo ovviamente le classi in cui non c'erano debiti) se ci fosse pericolo ad effettuare la procedura senza un backup corrente. In caso come potrei risolvere ? Grazie mille.
@trinko

**Comment by luigiadoc** [comment]
ho tentato anche con un registro scaricato in demo e NON effettuando gli scrutini mi elimina automaticamente tutti gli studenti. Avendo invece nel registro scolastico effettuato gli scrutini del primo quadrimestre, secondo e del giudizio sospeso fatta eccezione per le classi nelle quali non sono stati attribuiti non dovrei avere problemi come accaduto nel demo ? Grazie anticipatamente.

**Reply by trinko** [reply]
> ho tentato anche con un registro scaricato in demo e NON effettuando gli scrutini mi elimina automaticamente tutti gli studenti. Avendo invece nel registro scolastico effettuato gli scrutini del primo quadrimestre, secondo e del giudizio sospeso fatta eccezione per le classi nelle quali non sono stati attribuiti non dovrei avere problemi come accaduto nel demo ? Grazie anticipatamente.

Non credo ti abbia eliminato gli alunni: vengono eliminati solo quelli disabilitati e quelli ammessi nella quinta. Probabilmente non ha trovato la classe dove spostare i promossi e li ha assegnato una classe nulla. Lo puoi controllare dal menu **ALUNNI > MODIFICA** utlizzando il filtro per classi.

Comunque il backup è sempre bene averlo. Consiglio di farlo a mano:
- per i file si usa FileZilla, ci vuole un po' ma scarichi tutto sul tuo PC e non rischi di perdere dati;
- per il database usi l'applicazione _phpMyAdmin_, che è accessibile sulla pagina di gestione del sito Altervista.Con questa applicazione, si può esportare l’intero database in un file, che verrà scaricato sul proprio computer. In caso di necessità si può usare il file scaricato per recuperare l'intero database.

**Reply by luigiadoc** [reply]
Va bene ! Avendo però effettuato tutti gli scrutini non dovrei avere problemi giusto ?

**Reply by trinko** [reply]
> Va bene ! Avendo però effettuato tutti gli scrutini non dovrei avere problemi giusto ?

Non dovresti. Ma consiglio avere sempre il backup di sicurezza pronto.
